README.md

```diff
@@ -130,3 +131,51 @@ for {
     }
 }
 ```
+
+### Disabling the Server Watch
```

Comment:
I wonder if this section should be called something like "Servers behind a load balancer" so folks know they need to read it? 😄